backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/process/IADProcessDAO.java

```diff
@@ -127,4 +127,6 @@ default void registerTableProcess(final String tableName, final AdProcessId adPr
 	void updateColumnNameByAdElementId(
 			@NonNull AdElementId adElementId,
 			@Nullable String newColumnName);
+
+	boolean isJasperJSONProcess(@NonNull AdProcessId processId);
 }
```

Comment:
i think it would be better to have retrieveProcessType that returns the enum. otherwise the next person will add a method isJasperSQLProcess and so on..